.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing
+
+## Process/Workflow
+
+The project's [source code](https://github.com/inclusive-design/baby-bliss-bot) is hosted on GitHub. All of the code
+that is included in a release lives in the main branch. The intention is that the main branch is always in a working
+state.
+
+This project uses a workflow where contributors fork the project repository, work in a branch created off of main,
+and submit a Pull Request against the project repo's main branch.
+
+### Issue Tracker
+
+Issues are tracked using [GitHub Issues](https://github.com/inclusive-design/baby-bliss-bot/issues). When creating a
+new issue, please pick the most appropriate type (e.g. bug, feature) and fill out the template with all the
+necessary information. Issues should be meaningful and describe the task, bug, or feature in a way that can be
+understood by the community. Opaque or general descriptions should be avoided. If you have a large task that will
+involve a number of substantial commits, consider breaking it up into subtasks.
+
+### Linting
+
+In order to avoid errors and common pitfalls in python language, all code should be regularly checked using the
+provided lint task.
+
+```bash
+flake8
+```
+
+### Pull Requests and Merging
+
+If you are starting work on a new feature or bug fix, create a new branch from [dev][dev]:
```

Comment:
Should this be ```main``` rather than ```dev```? I assume you mean the default branch here, but I might be mistaken.